modules/api-svc/src/lib/model/InboundTransfersModel.js

```diff
@@ -515,7 +515,17 @@ class InboundTransfersModel {
             await this.saveFxState();
 
             const internalRequest = shared.mojaloopFxQuoteRequestToInternal(body);
-
+            
+            // Check if transactionRequestId exists in cache 
+            if(body.transactionRequestId) {
```

Comment:
but we don't have `transactionRequestId` in _POST /fxQuotes_ payload according Swagger docs